### ISSUE TITLE
Use django-anymail

### ISSUE THREAD

**Issue by fjsj** [vintasoftware/django-react-boilerplate#635]

**Comment by vishu1089** [comment]
is this sttill open?

**Comment by fjsj** [comment]
Yes, please feel free to open a PR introducing django-anymail to the boilerplate. Make sure to update the README as well.

**Comment by vishu1089** [comment]
hey, please review this PR

https://github.com/vintasoftware/django-react-boilerplate/pull/667

**Comment by vaishnavi-2901** [comment]
please close this issue if done?
